module/VuFind/src/VuFind/Auth/LoginToken.php

```diff
@@ -229,11 +239,15 @@ public function deleteActiveToken()
     public function sendLoginTokenWarningEmail(\VuFind\Db\Row\User $user)
     {
         if (!empty($user->email)) {
+            $message = $this->viewRenderer->render('Email/login-warning.phtml', ['title' => $this->config->Site->title]);
```

Comment:
I don't see the Email/login-warning.phtml file. Is it possible it didn't get committed with the rest of the change?

Reply:
That was indeed the case